### DISCUSSION TITLE
Disable "Export is done!" dialog

### DISCUSSION THREAD

**Discussion by brunompicinini** [mifi/lossless-cut#1625] [Support] [answered]
Is it possible? I'd prefer if it wouldn't show :) Sorry if this has been answered somewhere else. I've tried searching for it, but couldn't find an answer.

**Comment by mifi** [comment]
check settings

**Reply by brunompicinini** [reply]
I did actually. More than once. It it's there, it's written in a way I can't figure it out :(

It's not "Show informational notifications" right?

And I'm not talking about the dialog BEFORE exporting (that one I got it), but the one right AFTER, which say "export is done!"

BTW, just to make sure, awesome app! Truly love it! Fast and simple :-)

**Reply by mifi** [reply, marked as answer]
> It's not "Show informational notifications" right?

Right. it's that setting. But I realized there's a bug in 3.54.0 where that button doesn't work: https://github.com/mifi/lossless-cut/issues/1541

you can try this version instead: https://github.com/mifi/lossless-cut/releases/tag/v3.55.2

**Reply by brunompicinini** [reply]
ohhh I see that's why. Cool will try that :)

BTW: just donated to the project! Love how fast it is.

**Reply by mifi** [reply]
thanks a lot!
